eng/scripts/Language-Settings.ps1

```diff
@@ -308,11 +308,11 @@ function PackageDependenciesResolve($artifactNamePrefix, $packageDirectory) {
   $artifactDownloadOutput = mvn `
     dependency:copy `
     -Dartifact="$pomArtifactName" `
-    -DoutputDirectory="$packageDirectory"
+    -DoutputDirectory="$packageDirectory" | Out-Null
 
   if ($LASTEXITCODE) {
     LogWarning "Could not download pom artifact: $pomArtifactName"
-    $artifactDownloadOutput | Write-Host
+    $artifactDownloadOutput | Out-String | Write-Host
```

Comment:
Will this ever have a value based on your changes above?

Reply:
The correct fix is here: Azure/azure-sdk-tools#1977